Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -355,8 +363,80 @@ private void RenderDataCell (BocColumnRenderingContext<TBocColumnDefinition> ren
 
       renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
 
+      var validationInfoText = renderingContext.Control.GetResourceManager().GetString(BocList.ResourceIdentifier.ValidationErrorInfoCell);
+      var validationFailures = renderingContext.Control.ValidationFailureRepository.GetUnhandledValidationFailuresForDataCell(
+          arguments.BusinessObject,
+          renderingContext.ColumnDefinition,
+          false);
+
+      if (validationFailures.Count > 0)
+      {
+        var tooltipStringBuilder = new StringBuilder();
+        tooltipStringBuilder.AppendLine(validationInfoText);
+        foreach (var validationFailure in validationFailures)
+        {
+          tooltipStringBuilder.Append(" • ");
```

Comment:
```suggestion
          tooltipStringBuilder.Append("• ");
```

gut feeling: we don't need the left space.